scripts/src/javatests/org/oppia/android/scripts/regex/RegexPatternValidationCheckTest.kt

```diff
@@ -2153,6 +2158,98 @@ class RegexPatternValidationCheckTest {
       )
   }
 
+  fun testFileContent_xmlLayouts_includesNonColorComponentReferences_fileContentIsNotCorrect() {
```

Comment:
This & the test below are missing ``@Test`` annotations. Just to check, did you verify that these tests fail when the new checks are temporarily removed?

Reply:
Yes Ben, that are failing both in my local as well as in GitHub workflow.